src/_scss/pages/glossary/_animations.scss

```diff
@@ -1,4 +1,8 @@
 // slide in animation
+@import "../../lib/bourbon/css3/animation";
+@import "../../lib/bourbon/settings/prefixer";
+@import "../../lib/bourbon/addons/prefixer";
+
```

Comment:
This removed the horrible build times for the glossaryPage style sheet; just happened on another one once I fixed this though?